### PR TITLE
[13.0] [MIG] quality_control_mrp: Migration to v13

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT26774

Please @pedrobaeza @victoralmau review this

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This one should be renamed to `quality_control_mrp_oca` as well.

**Comment by CarlosRoca13** [comment]
@pedrobaeza name changed

File: quality_control_mrp_oca/models/qc_inspection.py [hunk 1]
```diff
@@ -31,18 +28,17 @@ def get_production(self):
...
-    @api.multi
     def object_selection_values(self):
-        objects = super(QcInspection, self).object_selection_values()
+        objects = super().object_selection_values()
         objects.append(("mrp.production", "Manufacturing Order"))
         return objects
 
     production_id = fields.Many2one(
-        comodel_name="mrp.production", compute="get_production", store=True
+        comodel_name="mrp.production", compute="_compute_production", store=True
```
Comment (pedrobaeza) [inline quality_control_mrp_oca/models/qc_inspection.py]:
```suggestion
        comodel_name="mrp.production", compute="_compute_production_id", store=True
```

File: quality_control_mrp_oca/views/mrp_production_view.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
         <field name="inherit_id" ref="mrp.mrp_production_form_view" />
         <field
             name="groups_id"
-            eval="[(4, ref('quality_control.group_quality_control_user'))]"
+            eval="[(4, ref('quality_control_oca.group_quality_control_user'))]"
```
Comment (pedrobaeza) [inline quality_control_mrp_oca/views/mrp_production_view.xml:20]: I think you can remove the module part, as relative links may work here.

File: quality_control_mrp_oca/views/qc_inspection_view.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <record model="ir.ui.view" id="qc_inspection_form_view_production">
         <field name="name">qc.inspection.form.view.production</field>
         <field name="model">qc.inspection</field>
-        <field name="inherit_id" ref="quality_control.qc_inspection_form_view" />
+        <field name="inherit_id" ref="quality_control_oca.qc_inspection_form_view" />
```
Comment (pedrobaeza) [inline quality_control_mrp_oca/views/qc_inspection_view.xml:8]: I think you can remove the module part, as relative links may work here (the same for the rest)

**Comment by CarlosRoca13** [comment]
@pedrobaeza If I delete the module part It will reffer to `quality_control_mrp_oca` module and not to `quality_control_oca`, so It will not work. 

I changed the compute method name as you said

File: quality_control_mrp_oca/i18n/de.po [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr "Prüfungen"
+
+#~ msgid "Manufacturing Order"
+#~ msgstr "Fertigungsauftrag"
+
+#~ msgid "Product"
+#~ msgstr "Produkt"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/de.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/es.po [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr "inspecciones"
+
+#~ msgid "Manufacturing Order"
+#~ msgstr "Órden de producción"
+
+#~ msgid "Product"
+#~ msgstr "Producto"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/es.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/fr.po [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr "inspections"
+
+#~ msgid "Manufacturing Order"
+#~ msgstr "Ordre de fabrication"
+
+#~ msgid "Product"
+#~ msgstr "Article"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/fr.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/gl.po [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr ""
+
+#~ msgid "Manufacturing Order"
+#~ msgstr "Orde de fabricación"
+
+#~ msgid "Product"
+#~ msgstr "Produto"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/gl.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/hr.po [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+msgid "Quality inspections from production order"
+msgstr ""
+
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr ""
+
+#~ msgid "Product"
+#~ msgstr "Proizvod"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/hr.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/it.po [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr "ispezioni"
+
+#~ msgid "Manufacturing Order"
+#~ msgstr "Ordine di produzione"
+
+#~ msgid "Product"
+#~ msgstr "Prodotto "
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/it.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/pt.po [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr "inspeções"
+
+#~ msgid "Manufacturing Order"
+#~ msgstr "Ordem de Fabrico"
+
+#~ msgid "Product"
+#~ msgstr "Produto"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/pt.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/pt_BR.po [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr "inspeções"
+
+#~ msgid "Manufacturing Order"
+#~ msgstr "Ordem de Produção"
+
+#~ msgid "Product"
+#~ msgstr "Produto"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/pt_BR.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/ro.po [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+msgid "Quality inspections from production order"
+msgstr ""
+
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr ""
+
+#~ msgid "Product"
+#~ msgstr "Produs"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/ro.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/sl.po [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr "inšpekcije"
+
+#~ msgid "Manufacturing Order"
+#~ msgstr "Proizvodni nalog"
+
+#~ msgid "Product"
+#~ msgstr "Proizvod"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/sl.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/tr.po [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+msgid "Quality inspections from production order"
+msgstr ""
+
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr ""
+
+#~ msgid "Product"
+#~ msgstr "Ürün"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/tr.po]:
```suggestion
```

File: quality_control_mrp_oca/i18n/vi_VN.po [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+msgid "Quality inspections from production order"
+msgstr ""
+
+#. module: quality_control_mrp_oca
+#: model_terms:ir.ui.view,arch_db:quality_control_mrp_oca.mrp_production_qc_view
+msgid "inspections"
+msgstr ""
+
+#~ msgid "Product"
+#~ msgstr "Sản phẩm"
```
Comment (victoralmau) [inline quality_control_mrp_oca/i18n/vi_VN.po]:
```suggestion
```

**Review by victoralmau — APPROVED** [review]
Good job, only some non-blocking changes

File: quality_control_mrp_oca/views/mrp_production_view.xml [hunk 16]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2018 Simone Rubino - Agile Business Group
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="action_qc_inspection_per_production" model="ir.actions.act_window">
+        <field name="name">Quality inspections from production order</field>
+        <field name="res_model">qc.inspection</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[('production_id', '=', active_id)]</field>
```
Comment (enriquemartin) [inline quality_control_mrp_oca/views/mrp_production_view.xml:9]: here we can set a context for default production value. Maybe something like this:
`<field name="context">{'default_object_id': 'mrp.production,' + str(active_id)}</field>`
or add the same context to the button that calls this action?

**Review by enriquemartin — APPROVED** [review]
added some comments, but LGTM

**Comment by pedrobaeza** [comment]
@CarlosRoca13 can you implement that thing for merging?

**Comment by CarlosRoca13** [comment]
@enriquemartin added the suggested change

**Pull request merged**